contents/handbook/growth/sales/new-sales.md

```diff
@@ -0,0 +1,141 @@
+---
+title: New sales
+sidebar: Handbook
+showTitle: true
+---
+
+This is our playbook for managing new sales. These are customers who have contacted sales proactively - we don't do outbound sales. Some large customers have a very prescriptive internal process that you will need to follow, but for the vast majority of our customers you will be the one driving process (and even for customers with a defined process, don't just sit back and be a passenger!).
+
+## Sales process
+
+This is an overview for what you should actually be doing with a customer at each stage of the sales process. For details on how to log this in HubSpot specifically, visit the relevant [HubSpot flow](/handbook/growth/sales/crm#inbound-hands-on-pipeline). 
+
+**1. People contact us**
+
+Most people fill in the [contact sales](/contact-sales) form via the website. Some email sales@posthog.com or raise a Zendesk ticket.
+
+**2. We assign and qualify**
+
+Cameron reviews these and decides whether to route to [self-serve](/handbook/growth/sales/crm#inbound-self-serve-pipeline), schedule a demo, or pass to Simon (if they seem Very Large).  This handoff is currently a gut feel based on company size/revenue as Very Large customers may technically be considered low ICP per our [scoring](/handbook/growth/sales/icp). 
+
+**3. Initial demo**
+
+Some general principles:
+- Be genuinely helpful - our customers usually already have specific problems in mind, so help them solve those first rather 
+- Get to the point - our ICP is technical and particularly sensitive to sales BS
+- Don't [promise](/handbook/growth/sales/overview#enterprise-customers) things we can't/don't want to deliver
+- Get into the actual demo as quickly as possible - you should spend most of your time here, not presenting PostHog's vision
+
+We have a simple deck (seriously, it's like 5 slides) - ask someone on the Sales team for an invite to our Pitch account so you can create customized versions. 
+
+You generally want to make sure you cover:
+
+- Introduction, who is there, and what their roles are
+- Why they are here
+- Opportunity for them to ask up-front questions and get direct answers to them
+- Demo product features according to what they need
+- Consider complementary products, e.g. analytics + replay; analytics + warehouse; surveys + flags
+- More Q&A
+- Pricing - try and understand what their likely usage and spend is, as well as willingness to spend / commit to an annual plan
+- Next steps - this is really important, don't just end the demo with no clear follow up action
```

Comment:
@simfish85 @camerondeleone @minekansu Let me know if there are any slides or topics you regularly use and are missing too. 

My usual approach for this is to create a small library of slides that we can stick in an appendix at the back for you to use if helpful. If you know a lead is using LaunchDarkly, for example, you can simply copy/paste it in.

Reply:
Maybe we could have a 'Why PostHog' slide at the start before we talk through product?  Even better if we can include a customer quote on why they moved to us.

In addition, if people ask me why PostHog vs Amplitude or Launchdarkly (say) it's normally at the end a of a demo so I just talk through consolidation of tools, as I don't think there is anything mindblowing if you do a point to point comparison of the specific product areas.  So I don't think any more slides are needed there.

Finally we could maybe have something updated which covers teams/enterprise etc

Reply:
Great. Can you share the deck with me, when you get a minute? I'll take a look.

Reply:
Are you in our Pitch account?  [Here](https://app.pitch.com/app/presentation/d1a6a499-5bd9-485d-925a-d274f939503b/4bf9bc6f-c37b-4783-9323-3d23e9313f66) if so

Reply:
Great, I've made some tweaks. Nothing dramatic, as I've never seen this used in demos myself, but:

- Added notes about EU/US hosting
- Addressed some small layout issues
- Relaid out the pricing table to be clearer
- Relaid out the Self Serve/Enterprise pages to be clearer
- Added a page for creds, but kept it simple. One quote, and point to further references. 
- Moved replaced slides to the end. If you don't like the new ones, easy to roll back.